### ISSUE TITLE
Hide history button on models

### ISSUE THREAD

**Issue by Mte90** [unfoldadmin/django-unfold#185]
Right now there isn't an option to not print that button on models if the feature is unused.

https://github.com/unfoldadmin/django-unfold/blob/6772bcf19735041e161c1d56395482604f8bedc7/src/unfold/templates/admin/change_form_object_tools.html#L6

**Comment by lukasvinclav** [comment]
Is it possible to disable this link in Django admin?

**Comment by Mte90** [comment]
I am not sure but as this projects has custom parameters I think that is something easy to do.

**Comment by justinqian42** [comment]
As of today, there doesn't seem to be a native feature in Django Admin to hide the history button. However, with the latest unfold templates, you can easily rewrite templates/admin/change_form_object_tools.html to achieve that.

**Comment by Mte90** [comment]
Yes it is possible but if history is not enabled in the model clicking that button will show that error so maybe this dashboard can improve that UX automatically checking if the history is enabled/exist and add the button.

**Comment by justinqian42** [comment]
Easy thing.

Add one attribute in your ModelAdmin class, like.

class ProductModelAdmin(ModelAdmin):
    display_history_button = False

Overwrite templates/admin/change_form_object_tools.html as below.

from 


    {% trans 'History' as title %}
    {% add_preserved_filters history_url as link %}
    {% include "unfold/helpers/tab_action.html" with title=title link=link %}


to

    {% if display_history_button %}
        {% trans 'History' as title %}
        {% add_preserved_filters history_url as link %}
        {% include "unfold/helpers/tab_action.html" with title=title link=link %}
    {% endif %}


Also, modify changeform_view method, update 

        extra_context.update(
            {
                "actions_submit_line": self.get_actions_submit_line(request),
                "actions_detail": actions,
            }
        )

to

        extra_context.update(
            {
                "actions_submit_line": self.get_actions_submit_line(request),
                "actions_detail": actions,
                "display_history_button: self.display_history_button,
            }
        )


If you wanna achieve object-level history button display on/off, add in BooleanField display_history_button in your model, and change the above codes to

        extra_context.update(
            {
                "actions_submit_line": self.get_actions_submit_line(request),
                "actions_detail": actions,
                "display_history_button: YourModel.objects.get(id=object_id).display_history_button,
            }
        )



These codes haven't been verified. Use them at your own risk. But I think they should work.
